__tests__/endpoint.tests/endpoint.test.js

```diff
@@ -376,3 +376,47 @@ describe("articles", () => {
     });
   });
 });
+
+describe("comments", () => {
+  describe("Delete comment by Id", () => {
+    test("DELETE:200 deletes a comment and returns the deleted comment", () => {
+      return request(app)
+        .delete("/api/comments/1")
+        .expect(200)
```

Comment:
a delete returns a 204 and you dont need to expect anything else except the status code